addons/binding/org.openhab.binding.hdpowerview/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,30 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: Hunter Douglas PowerView Binding
+Bundle-SymbolicName: org.openhab.binding.hdpowerview;singleton:=true
+Bundle-Vendor: openHAB
+Bundle-Version: 2.0.0.qualifier
+Bundle-RequiredExecutionEnvironment: JavaSE-1.7
+Bundle-ClassPath: .,
+ lib/jcifs-1.3.18.jar
```

Comment:
> Will the LGPL license preclude its inclusion in OH as well?

No, in OH we can live with it. But I think that the NetBIOS name lookup is useful in other bindings as well (@kgoderis, weren't you also checking for that manually somewhere?), so instead of including this functionality in a binding, it would rather want to have it in the framework to be shared. And for this, ESH would be the best place to put it. Alternatively, I would put it in openhab-core, but this is only second best...